### ISSUE TITLE
Add month view option for the calendar.

### ISSUE THREAD

**Issue by sojitrajay** [wayfinders/WRCalendarView#10]
I want to see month wise event listing in the calendar which is not in this application yet.

Can we implement this into this library?

**Comment by Naveentp** [comment]
Yes. Even I am looking for this implementation. @wayfinders Please implement the month and Year feature also.
